tutorial/consuming_packages/different_configurations.rst

```diff
@@ -46,7 +46,9 @@ folder:
 
 As you can see, the profile has different sections. The ``[settings]`` section is the one
 that has information about things like the operating system, architecture, compiler, and
-build configuration. When you call a Conan command setting the ``--profile`` argument,
+build configuration. The default profile can also be checked with the command :command:`conan profile show`.
+
```

Comment:
Maybe we could move this to the comments in the code snippet, so that users can try also the command at the same moment that they output the profile to the console
```suggestion
build configuration.
```

Reply:
done!